src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordGetHashCode.cs

```diff
@@ -83,6 +97,12 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 diagnostics.Add(ex.Diagnostic);
                 F.CloseMethod(F.ThrowNull());
             }
+
+            static void ensureEqualityComparerHelpers(SyntheticBoundNodeFactory F, ref MethodSymbol? equalityComparer_GetHashCode, ref MethodSymbol? equalityComparer_get_Default)
+            {
+                equalityComparer_GetHashCode ??= F.WellKnownMethod(WellKnownMember.System_Collections_Generic_EqualityComparer_T__GetHashCode, isOptional: false)!;
```

Comment:
Consider using the `MethodSymbol WellKnownMethod(WellKnownMember)` overload, which passes `isOptional: false` and suppresses the nullable warning internally.